### ISSUE TITLE
 Parsing of show slave status in MySQL 5.7 is failing

### ISSUE THREAD

**Issue by ryuichi1208** [mackerelio/mackerel-plugin-mysql#7]
#### What happened?

When I run this plugin on MySQL 5.7, I get the following error when parsing ```show slave status``` in [fetchShowSlaveStatus()](https://github.com/mackerelio/mackerel-plugin-mysql/blob/v1.1.0/lib/mysql.go#L298-L323).

#### To reproduce

* Plugin Version: v1.1.0
* MySQL Version: 5.7.33

```
$ ./mackerel-plugin-mysql -host testhost
2023/02/09 13:21:10 OutputValues:  FetchMetrics (Slave Status): sql: expected 57 destination arguments in Scan, not 2
```

**Comment by takaidohigasi** [comment]
mackerel-agent-plugin after v0.75.0 seems to be affected

**Comment by lufia** [comment]
@ryuichi1208 Sorry we noticed this issue today. We will fix this (ref #8)

**Comment by lufia** [comment]
We released fixed version of this plugin, mackerel-agent-plugins v0.77.0, today.
